services/common/src/components/projects/projectUtils.spec.ts

```diff
@@ -9,8 +9,8 @@ const TEST_PARAMETERS = [
         testFunction: areFieldsDisabled,
         coreDisabledStatuses: ["WDN", "COM"],
         coreEnabledStatuses: ["DFT", "SUB", "ASG", "UNR", "CHR", "OHD"],
-        msDisabledStatuses: ["SUB", "ASG", "UNR", "WDN", "OHD", "COM"],
-        msEnabledStatuses: ["DFT", "CHR"],
+        msDisabledStatuses: ["SUB", "UNR", "WDN", "OHD", "COM"],
+        msEnabledStatuses: ["DFT", "CHR", "ASG"],
```

Comment:
I do find it confusing that Assigned was added to enabled- like, generally they're still not supposed to be able to edit when it's in Assigned status, right? Is the status even assigned when there's no project lead assigned?

Reply:
Hmm, so is the flow that's causing issues basically:
1. an MS/CORE user makes a draft project summary
2. they assign a project lead
3. the project gets put in Assigned status even though it's not submitted

I guess it's pretty wacky! I'm thinking that for the tests though, maybe pass a parameter to indicate that it _has_ been submitted so that we know what to expect generally, and then add another test for the special case where it's assigned but not submitted?

Reply:
I reverted the change to the arrays, and a new test has been added now.